charts/mlrun-ce/templates/NOTES.txt

```diff
@@ -1,8 +1,10 @@
+{{- if or (.Values.jupyterNotebook.enabled) (.Values.mlrun.enabled) (.Values.minio.enabled) (.Values.pipelines.enabled) ( index .Values "kube-prometheus-stack" "enabled")}}
 You're up and running !
-
+{{- if .Values.jupyterNotebook.enabled }}
 1. Jupyter UI is available at:
   http://{{ .Values.global.externalHostAddress }}:{{ .Values.jupyterNotebook.service.nodePort }}
-
+{{- end}}
+{{- if .Values.mlrun.enabled }}
 2. Nuclio UI is available at:
   http://{{ .Values.global.externalHostAddress }}:{{ .Values.global.nuclio.dashboard.nodePort }}
```

Comment:
I'm guessing you want a `{{- if .Values.nuclio.enabled }}` here and a the mlrun check a couple of lines down no?